### PR TITLE
Platen is so on brand now

### PR DESCRIPTION
**Pull request by Z33**
- Official logo added
- Header now centered
- Book meta now fetched from index.mdx
- Library sec updated
- Page headers component added
- Book covers now have an overlay
- Some other small layout updates
- Added Book Template

### CODE REVIEW

File: content/books/iron-flax-bone/index.mdx [hunk 1]
```diff
@@ -13,13 +13,16 @@ tags:
   - pamphlet
 features:
   - d12 system
+meta:
+  chapters: 24
```
Comment (michaeltlombardi) [inline content/books/iron-flax-bone/index.mdx]:
```suggestion
  chapters: 4
```

File: content/books/iron-flax-bone/index.mdx [hunk 2]
```diff
@@ -13,13 +13,16 @@ tags:
...
+  pages: 248
```
Comment (michaeltlombardi) [inline content/books/iron-flax-bone/index.mdx]:
```suggestion
  pages: 3
```

I'm not actually sure if we want/need this vs a "time to read" metric? 🤔

File: content/books/iron-flax-bone/index.mdx [hunk 3]
```diff
@@ -13,13 +13,16 @@ tags:
...
+  ages: 12+
+  language: English
 ---
 
 # Iron, Flax, & Bone
 
-> _**Iron** for the sword and hoe,_ <br />
-> _**Flax** for crops and string of bow,_ <br />
-> _**Bone** to bend and break in woe._
+> _**Iron** for the sword and hoe,_ <br /> > _**Flax** for crops and string of bow,_ <br /> > _**Bone** to bend and break in woe._
```
Comment (michaeltlombardi) [inline content/books/iron-flax-bone/index.mdx]: What happened here? 😮

File: content/books/platen/index.mdx [hunk 4]
```diff
@@ -10,6 +10,11 @@ features:
   - Feature 1
   - Feature 2
   - Feature 3
+meta:
```
Comment (michaeltlombardi) [inline content/books/platen/index.mdx:13]: What happens if the meta is empty or nonexistant?

File: content/books/iron-flax-bone/index.mdx (hunk 2 above)
Comment (Z33) [reply content/books/iron-flax-bone/index.mdx]: All just random meta. Plan to try and make some dynamic data easy to access for Authors, but until then it'll be static and up to them what they showcase here.

File: content/books/platen/index.mdx (hunk 4 above)
Comment (Z33) [reply content/books/platen/index.mdx:13]: Nothing will render since it's considered to be null

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**